### ISSUE TITLE
New Buefy install on brand new Vuejs webpack project is broken

### ISSUE THREAD

**Issue by mrunkel** [rafaelpimpa/buefy#111]
### Overview of the problem

**Buefy** version: 0.4.2
**Vuejs** version: 2.3.4
**OS/Browser**: OS X

### Description

Buefy doesn't work when installing a new webpack or webpack simple skeleton with vue-cli

### Steps to reproduce

1. ```vue init webpack webpack```
2. npm install
3. npm install --save buefy
4. Modify main.js per install instructions
```javascript
    import Buefy from 'buefy'
    import 'buefy/lib/buefy.css'
    Vue.use(Buefy)```
5. run ```npm run dev```

### Expected behavior

I expect the sample web page to load.

### Actual behavior

```
 ERROR  Failed to compile with 1 errors                                                                                                                                                                                                            3:04:46 PM

This dependency was not found:

* buefy in ./src/main.js

To install it, you can run: npm install --save buefy```

running npm install --save buefy does nothing.

output from webpack --display-error-details

```ERROR in ./src/main.js
Module not found: Error: Can't resolve 'buefy' in '/Volumes/ssd/simple/src'
resolve 'buefy' in '/Volumes/ssd/simple/src'
  Parsed request is a module
  using description file: /Volumes/ssd/simple/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Volumes/ssd/simple/package.json (relative path: ./src)
    resolve as module
      /Volumes/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      /Volumes/ssd/simple/src/node_modules doesn't exist or is not a directory
      looking for modules in /Volumes/ssd/node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
        No description file found
        as directory
          /Volumes/ssd/node_modules/buefy doesn't exist
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          /Volumes/ssd/node_modules/buefy doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          /Volumes/ssd/node_modules/buefy.js doesn't exist
        .json
          Field 'browser' doesn't contain a valid alias configuration
          /Volumes/ssd/node_modules/buefy.json doesn't exist
      looking for modules in /Volumes/ssd/simple/node_modules
        using description file: /Volumes/ssd/simple/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Volumes/ssd/simple/package.json (relative path: ./node_modules)
          using description file: /Volumes/ssd/simple/node_modules/buefy/package.json (relative path: .)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Volumes/ssd/simple/node_modules/buefy is not a file
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Volumes/ssd/simple/node_modules/buefy.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Volumes/ssd/simple/node_modules/buefy.json doesn't exist
            as directory
              existing directory
                using path: /Volumes/ssd/simple/node_modules/buefy/index
                  using description file: /Volumes/ssd/simple/node_modules/buefy/package.json (relative path: ./index)
                    no extension
                      Field 'browser' doesn't contain a valid alias configuration
                      /Volumes/ssd/simple/node_modules/buefy/index doesn't exist
                    .js
                      Field 'browser' doesn't contain a valid alias configuration
                      /Volumes/ssd/simple/node_modules/buefy/index.js doesn't exist
                    .json
                      Field 'browser' doesn't contain a valid alias configuration
                      /Volumes/ssd/simple/node_modules/buefy/index.json doesn't exist
                use ./lib/src/index.js from main in package.json
                  using description file: /Volumes/ssd/simple/node_modules/buefy/package.json (relative path: .)
                    Field 'browser' doesn't contain a valid alias configuration
                  after using description file: /Volumes/ssd/simple/node_modules/buefy/package.json (relative path: .)
                    using description file: /Volumes/ssd/simple/node_modules/buefy/package.json (relative path: ./lib/src/index.js)
                      as directory
                        /Volumes/ssd/simple/node_modules/buefy/lib/src/index.js doesn't exist
                      no extension
                        Field 'browser' doesn't contain a valid alias configuration
                        /Volumes/ssd/simple/node_modules/buefy/lib/src/index.js doesn't exist
                      .js
                        Field 'browser' doesn't contain a valid alias configuration
                        /Volumes/ssd/simple/node_modules/buefy/lib/src/index.js.js doesn't exist
                      .json
                        Field 'browser' doesn't contain a valid alias configuration
                        /Volumes/ssd/simple/node_modules/buefy/lib/src/index.js.json doesn't exist
[/Volumes/node_modules]
[/node_modules]
[/Volumes/ssd/node_modules/package.json]
[/Volumes/ssd/simple/src/node_modules]
[/Volumes/ssd/node_modules/buefy/package.json]
[/Volumes/ssd/node_modules/buefy]
[/Volumes/ssd/node_modules/buefy]
[/Volumes/ssd/node_modules/buefy.js]
[/Volumes/ssd/node_modules/buefy.json]
[/Volumes/ssd/simple/node_modules/buefy]
[/Volumes/ssd/simple/node_modules/buefy.js]
[/Volumes/ssd/simple/node_modules/buefy.json]
[/Volumes/ssd/simple/node_modules/buefy/index]
[/Volumes/ssd/simple/node_modules/buefy/index.js]
[/Volumes/ssd/simple/node_modules/buefy/index.json]
[/Volumes/ssd/simple/node_modules/buefy/lib/src/index.js]
[/Volumes/ssd/simple/node_modules/buefy/lib/src/index.js]
[/Volumes/ssd/simple/node_modules/buefy/lib/src/index.js.js]
[/Volumes/ssd/simple/node_modules/buefy/lib/src/index.js.json]
 @ ./src/main.js 3:0-26

ERROR in ./~/buefy/lib/buefy.css
Module parse failed: /Volumes/ssd/simple/node_modules/buefy/lib/buefy.css Unexpected character '@' (2:0)
You may need an appropriate loader to handle this file type.
| /*! bulma.io v0.4.2 | MIT License | github.com/jgthms/bulma */
| @-webkit-keyframes spinAround {
|   from {
|     -webkit-transform: rotate(0deg);
 @ ./src/main.js 4:0-29```

**Comment by rafaelpimpa** [comment]
Looking at it.

**Comment by rafaelpimpa** [comment]
Looks like ``vue build`` don't create the ``src`` folder anymore, publishing v0.4.2 right now.

**Comment by mrunkel** [comment]
Thanks for the quick response!

**Comment by mrunkel** [comment]
@rafaelpimpa So, that problem is fixed and it works for the webpack template from vue-cli.

FYI, there is something else broken in the webpack-simple template for buefy.   In order to avoid the error:
```ERROR in ./~/buefy/lib/buefy.css
Module parse failed: /Volumes/ssd/simple/node_modules/buefy/lib/buefy.css Unexpected character '@' (2:0)
You may need an appropriate loader to handle this file type.
| /*! bulma.io v0.4.2 | MIT License | github.com/jgthms/bulma */
| @-webkit-keyframes spinAround {
|   from {
|     -webkit-transform: rotate(0deg);
 @ ./src/main.js 4:0-29
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/main.js```

You have to:
1. ```npm install --save style-loader```
2. Add the following to webpack.config.js
```      {
        test: /\.css$/,
        loader: 'style-loader!css-loader'
      }```
under rules:

I don't know if that should be a buefy issue (doc issue maybe?) or a vue-cli issue.   Let me know if you think I should file it over there.

**Comment by rafaelpimpa** [comment]
That's because you're importing the css in the .js file, I'll add a observation later on, thank you :)

**Comment by ashwanizsktm** [comment]
I am having the same issue I have tried my wepack configuring mmultiple times but it doesn't resolve, can anyone help me with this.

**Comment by mrunkel** [comment]
Commenting me too, on a ticket that was closed 4.5 years ago isn’t going to get you far.  Everything has changed since then.  File a new issue.
